connectionpool.go

```diff
@@ -439,19 +439,14 @@ func (pool *hostConnPool) fill() {
 
 		// filled one
 		fillCount--
-		// connect all remaining connections to this host
```

Comment:
The reason is to reduce gocql startup time. There were conflicting comments in that code, so I restored the original behavior because I thought opening a single connection synchronously was a good balance (it gives synchronous feedback to user, but doesn't make app wait for all n connections to be established).

The previous code did not really handle errors differently (connectMany() simply logs errors).